cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -341,19 +369,21 @@
             display: none;
         }
 
-        &_content {
+        // 1st-level menu - Desktop.
+        &_content-1 {
+            &[aria-expanded="true"] {
+                .o-mega-menu_content-1-wrapper {
+                    display: block;
+                    transform: translateY( 0 );
+                }
+            }
             &-list {
-                padding-left: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
+                //padding-left: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/24de85959a5034e9a6a6dbdd133f83001df30f36